lib/bundle.js

```diff
@@ -76,6 +77,101 @@ function extractBundleName(fileName) {
   return path.relative(config.pjson.baseURL, fileName.replace(/\.js$/, '')).replace(/\\/g, '/');
 }
 
+function getModuleName (file) {
```

Comment:
@guybedford I am not completely sure about this implementation. Please take a look at it.